python/README.md

```diff
@@ -0,0 +1,62 @@
+# python
+
+A Wing library for integrating with Python code.
+
+## Prerequisites
+
+* [winglang](https://winglang.io).
+
+## Installation
+
+```sh
+npm i @winglibs/python
+```
+
+## Usage
+
+```js
+bring python;
+
+let func = new python.Function(
+  path: "./test-assets",
+  handler: "main.handler",
+);
+
+test "invokes the function" {
+  let res = func.invoke();
+}
+```
+
+It is also possible to interact with Wing resources through the python code
+
+```js
+// main.w
+let bucket = new cloud.Bucket();
+let func = new python.Function(
+  path: "./test-assets",
+  handler: "main.handler",
+);
+
+func.liftClient("bucket", bucket, ["get", "put"]);
+```
+
+```python
+# main.py
+from wing_helper import *
```

Comment:
```suggestion
from wing import *
```

The `_helper` is not adding any context (and actually "from wing import *" reads so nice).